src/CodeGenHelpers/PropertyBuilder.cs

```diff
@@ -33,6 +33,8 @@ internal PropertyBuilder(string name, Accessibility? accessModifier, ClassBuilde
             Class = builder;
         }
 
+        public DocumentationComment XmlDoc { get; } = new DocumentationComment();
```

Comment:
this should probably be private and only exposed via the helper methods on the PropertyBuilder.